spec/gateway/pca_address_list_spec.rb

```diff
@@ -0,0 +1,16 @@
+describe PostcodeValidation::Gateway::PCAAddressList do
+
+  context do
+    let(:search_term) { 'SE1 0SW' }
+    let(:country) { 'GB' }
+
+    it do
+      VCR.use_cassette('PCA-address-list') do
+        subject = described_class.new.query(search_term: search_term, country: country)
+
+        expect(subject.first.street_address).to eq('A J Wealth Management Ltd')
```

Comment:
^^ Did you randomly choose this company name? This looks like the company name for Adam (the guy who looks after our company pensions?)